pkg/portal/cluster/clusteroperators.go

```diff
@@ -27,15 +36,25 @@ func clusterOperatorsInformationFromOperatorList(operators *configv1.ClusterOper
 	for _, co := range operators.Items {
 		var Available = configv1.ConditionUnknown
```

Comment:
not from this pr, but I'm not sure why this is capitalized, that would mean that it is exported and is kind of confusing